tests/unit/test_gbq.py

```diff
@@ -246,6 +246,28 @@ def test_to_gbq_doesnt_run_query(
     mock_bigquery_client.query.assert_not_called()
 
 
+def test_to_gbq_uploading_empty_dataframe(
```

Comment:
Very reasonable to copy code also testing writes, but this is doing something slightly more involved re pandas versioning 

Could you copy the code here: https://github.com/pydata/pandas-gbq/blob/272aa7bebef5dc99869e7c44adf8011258c8d7c9/tests/system/test_gbq.py#L879 and put the test there?

Reply:
Hi @max-sixty thanks for your feedback. Two questions:

1. Shall I create a test method in `pandas-gbq/tests/system/test_gbq.py` named `test_upload_empty_data`, or just create an empty `pd.DataFrame` within `test_upload_data`? Which one is better?
2. Do I need to delete `test_to_gbq_uploading_empty_dataframe` in `pandas-gbq/tests/unit/test_gbq.py`?

Thanks!

Reply:
>     1. Shall I create a test method in `pandas-gbq/tests/system/test_gbq.py` named `test_upload_empty_data`,

This would be perfect! 
That's the only test that's needed (no need to keep the existing `test_to_gbq_uploading_empty_dataframe`)